app/src/main/java/org/simple/clinic/searchresultsview/SearchResultsItemType.kt

```diff
@@ -15,6 +15,33 @@ sealed class SearchResultsItemType<T : ViewHolder>(adapterId: Long) : GroupieIte
   companion object {
     private const val HEADER_NOT_IN_CURRENT_FACILITY = 0L
     private const val HEADER_NO_PATIENTS_IN_CURRENT_FACILITY = 1L
+
+    fun generateListItems(
```

Comment:
Can we have a more descriptive name. This sounds too generic and does not reveal intent.

Reply:
Yup, I'll think of something better.